packages/eslint-plugin/src/index.ts

```diff
@@ -3,17 +3,25 @@ import base from './configs/base';
 import eslintRecommended from './configs/eslint-recommended';
 import recommended from './configs/recommended';
 import recommendedRequiringTypeChecking from './configs/recommended-requiring-type-checking';
+import recommendedTypeChecked from './configs/recommended-type-checked';
 import strict from './configs/strict';
+import strictTypeChecked from './configs/strict-type-checked';
+import stylistic from './configs/stylistic';
+import stylisticTypeChecked from './configs/stylistic-type-checked';
 import rules from './rules';
 
 export = {
-  rules,
   configs: {
     all,
     base,
-    recommended,
     'eslint-recommended': eslintRecommended,
+    recommended,
     'recommended-requiring-type-checking': recommendedRequiringTypeChecking,
+    'recommended-type-checked': recommendedTypeChecked,
```

Comment:
i was going to say you could save that one file and do this instead:
```suggestion
    'recommended-requiring-type-checking': recommendedTypeChecked,
    'recommended-type-checked': recommendedTypeChecked,
```

Reply:
Oh ha yeah true. And on top of this & v6 overall already being breaking changes, with #6458 we don't have to worry about people reaching into `dist/`. Great.